tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+import pytest
+
+from tests.fixtures import bot_env, safe_environ
+
+__all__ = ("bot_env",)
+
+
+def pytest_configure(config: pytest.Config) -> None:  # noqa: ARG001
```

Comment:
Can't you `_` it instead?
```suggestion
def pytest_configure(_: pytest.Config) -> None:
```
```suggestion
def pytest_configure(_config: pytest.Config) -> None:
```

Reply:
Nope, `pytest` expects that the parameter is named `config`.

Reply:
Oh wow, okay.